duckduckgo/BrowserServicesKit
# PacketTunnelProvider crash simulation options

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1205376937712482/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1978
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1597
What kind of version bump will this require?: Minor

**Description**:

To verify that crash reporting works as expected from the PacketTunnelProvider extensions, this PR adds simulation options to trigger two types of crash: a fatal error and a memory overuse crash.

**Steps to test this PR**:
1. See macOS PR (link above)
2. See iOS PR (link above)

**OS Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
